### ISSUE TITLE
Other Scale types (enhancement)

### ISSUE THREAD

**Issue by Sermitse** [ersh1/DynamicCollisionAdjustment#1]
As far as i understand, your plugin checks for actor->GetScale(); only

I have a wip Giantess mod (im not the main author though) that edits the scale in 2 different ways, they can be chosen by the user:

1)        auto model = actor->Get3D(false);  
           float model_scale = model->local.scale; (Model scale method)

2)        string node_name = "NPC Root [Root]";
           auto node = find_node(actor, node_name, false);
           float node_scale = node->local.scale;  (Node scale method, we have own function for find_node)

The source if you need it: **https://github.com/QuantumEntangledAndy/GTS_Plugin/blob/spells/src/scale/modscale.cpp**

Basically im suggesting to make an optional MCM toggle that edits 'float scale = actor->GetScale()' to something like 'float scale = actor->GetScale() * node_scale * model_scale' to make our mods compatible.

p.s: if possible, if you'll want to mess with it further (or in general), it would be also nice to have compatibility option in Precision/Projectile Precision as well, it seems like our mods conflict currently: colliders are being reset by scale changes if we try to edit them on our end with Precision on.
